nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -23,14 +23,18 @@
 
 namespace nav2_waypoint_follower
 {
-
 WaypointFollower::WaypointFollower()
-: nav2_util::LifecycleNode("WaypointFollower", "", false)
+: nav2_util::LifecycleNode("WaypointFollower", "", false),
+  task_executor_at_waypoint_loader_("nav2_waypoint_follower",
+    "nav2_core::TaskExecutorAtWaypointArrival"),
+  default_task_executor_at_waypoint_id_{"task_executor_at_waypoint"},
+  default_task_executor_at_waypoint_type_{"nav2_waypoint_follower::WaitAtWaypointArrival"}
 {
   RCLCPP_INFO(get_logger(), "Creating");
 
   declare_parameter("stop_on_failure", true);
   declare_parameter("loop_rate", 20);
+  declare_parameter("task_executor_at_waypoint_plugin", default_task_executor_at_waypoint_id_);
```

Comment:
You can just set the default plugin here, you don't need to store it as a class variable. Please remove the `default_XYZ_` variables and just put them in the declares.